depends/packages/qt.mk

```diff
@@ -44,6 +44,7 @@ $(package)_config_opts += -no-ico
 $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-kms
 $(package)_config_opts += -no-linuxfb
+$(package)_config_opts += -no-libjpeg
```

Comment:
16441: missing changes in `build-aux/m4/bitcoin_qt.m4` - either too early or should be `partial` I guess.